### PR TITLE
Polishing some UI docs items that seem to be outdated

### LINKED ISSUE #34758: Updating JavaScript developer docs
### Description

Some of the docs seem to be oudated, and I'll try to update them.

### PR DESCRIPTION
**Pull request by ahus1**
Closes #34758

### CODE REVIEW

**Comment by ahus1** [comment]
@keycloak/ui - I tried to update the docs to my best knowledge after some help from the UI chat. Let me know if this looks ok.

File: js/apps/admin-ui/CONTRIBUTING.md
Comment (jonkoops) [inline js/apps/admin-ui/CONTRIBUTING.md:1]: The Node.js version stated here should be 22 or higher, that is the latest LTS version of Node.js.

File: js/apps/admin-ui/CONTRIBUTING.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ You can find out which version of Node.js you are using by running the following
 node --version
 ```
 
+For the `pnpm` command to be available, run the command `corepack enable` once.
```
Comment (jonkoops) [inline js/apps/admin-ui/CONTRIBUTING.md:17]: Perhaps we can simply copy this documentation that I wrote for one of my repos: https://github.com/jonkoops/kjs-playground?tab=readme-ov-file#setup

File: js/apps/admin-ui/CONTRIBUTING.md [hunk 2]
```diff
@@ -24,13 +26,13 @@ See the instructions in the [Keycloak server app](../keycloak-server/README.md).
 
 Now that the Keycloak sever is running it's time to run the development server for the Admin UI. This server is used to build the Admin UI in a manner that it can be iterated on quickly in a browser, using features such as [Hot Module Replacement (HMR)](https://vitejs.dev/guide/features.html#hot-module-replacement) and [Fast Refresh](https://www.npmjs.com/package/react-refresh).
 
-To start the development server run the following command:
+To start the development server run the following command from the root of the repository:
 
 ```bash
-pnpm dev
```
Comment (jonkoops) [inline js/apps/admin-ui/CONTRIBUTING.md:30]: This should not be changed, as running this command locally is a valid option.

File: js/apps/admin-ui/CONTRIBUTING.md [hunk 3]
```diff
@@ -56,16 +58,22 @@ Ensure the Keycloak and development server are running as [outlined previously](
 
 ### Running the tests
 
+Install cypress:
+
+```bash
+pnpm --filter @keycloak/keycloak-admin-ui exec cypress install
```
Comment (jonkoops) [inline js/apps/admin-ui/CONTRIBUTING.md:64]: This should happen automatically when running `pnpm install` I believe.

File: js/apps/admin-ui/CONTRIBUTING.md (hunk 1 above)
Comment (ahus1) [reply js/apps/admin-ui/CONTRIBUTING.md:17]: Hm, its beyond my skills to identify what to copy.

File: js/apps/admin-ui/CONTRIBUTING.md (hunk 2 above)
Comment (ahus1) [reply js/apps/admin-ui/CONTRIBUTING.md:30]: Why this one? What about the others?

**Comment by ahus1** [comment]
Sorry, too much half-knowledge on my end.

**Pull request closed without merging**